src/main/java/com/aws/Helper.java

```diff
@@ -99,8 +99,9 @@ static JSONObject getJsonObject(String message) {
                 exclusiveStartShardId = null;
             }
 
-        }while (exclusiveStartShardId != null);
-
+        }while (exclusiveStartShardId != null);*/
+         DescribeStreamResult describeStreamResult = kinesis.describeStream(describeStreamRequestWithExclusiveShardStart);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer

This code might not produce accurate results if the operation returns paginated results instead of all results. Consider adding another call to check for additional results.